tools/ImporterV2/ImporterV2.cs

```diff
@@ -0,0 +1,424 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Fhir.ImporterV2
+{
+    internal static class ImporterV2
```

Comment:
>ImporterV2 [](http://example.com/codeflow?start=26&length=10)

Please reflect usage of this tool in its name.
It uses $import, registers it, and monitors it progress.
I can see that it can fail in monitoring. Tool should provide option to monitor without registering.